components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementService.java

```diff
@@ -229,6 +229,18 @@ public abstract ServiceProvider getServiceProviderByClientId(String clientId, St
                                                                  String tenantDomain)
             throws IdentityApplicationManagementException;
 
+    /**
+     * Export Service Provider application with required attributes.
```

Comment:
if the `requiredAttributes` is empty we return the basic application details right ? shall we include that detail also here ?

Reply:
No. This will be triggered only if it has requiredAttributes not empty.